neuralmonkey/decoding_function.py

```diff
@@ -47,6 +48,12 @@ def attention_decoder(decoder_inputs, initial_state, attention_objects,
             outputs.append(output)
             states.append(state)
 
+        if summary_collections:
+            for i, a in enumerate(attention_objects):
+                alignments = tf.expand_dims(tf.transpose(tf.pack(a.attentions_in_time), perm=[1, 2, 0]), -1)
```

Comment:
This line is too long. We check the style of the files using `pylint`. If you run `tests/lint_run.sh`, it will check all python files get you a detailed report.